credentials/generate-revocation-set.py

```diff
@@ -91,6 +91,101 @@ def parse_vid_pid_from_distinguished_name(distinguished_name):
     return vid, pid
 
 
+def get_akid(cert: x509.Certificate) -> bytes:
+    try:
+        return cert.extensions.get_extension_for_oid(x509.OID_AUTHORITY_KEY_IDENTIFIER).value.key_identifier
+    except Exception:
+        return None
+
+
+def get_skid(cert: x509.Certificate) -> bytes:
+    try:
+        return cert.extensions.get_extension_for_oid(x509.OID_SUBJECT_KEY_IDENTIFIER).value.key_identifier
+    except Exception:
+        return None
```

Comment:
Please log in all exceptions what happened to help assist debugging.

Reply:
Logging the exceptions.